ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -35,8 +35,10 @@ type HeadGroupSpec struct {
 	Replicas *int32 `json:"replicas"`
 	// RayStartParams are the params of the start command: node-manager-port, object-store-memory, ...
 	RayStartParams map[string]string `json:"rayStartParams"`
-	// Template is the eaxct pod template used in K8s depoyments, statefulsets, etc.
+	// Template is the exact pod template used in K8s deployments, statefulsets, etc.
 	Template v1.PodTemplateSpec `json:"template"`
+	// RayResources is a string-int map specifying Ray custom resources and overrides for CPU,GPU,memory.
+	RayResources RayResources `json:"rayResources,omitempty"`
```

Comment:
Same question as comment at line 62 below (is it OK to use the same name RayResources for type and field)

Reply:
Oh, that is a little odd. It is done in a few places in this file already -- in the future, I might just stick to the underlying map type to make it simpler and avoid this bit of awkwardness.